### DISCUSSION TITLE
Auth to gcr.io and pkg.dev

### DISCUSSION THREAD

**Discussion by cjyar** [renovatebot/renovate#27344] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

My private GitHub repo contains kubernetes YAML files that reference private container images on gcr.io. I'm following the documentation at https://github.com/renovatebot/renovate/blob/main/docs/usage/docker.md#using-long-lived-service-account-credentials, but having no luck.

I've generated a `gcr.b64` file locally which contains the base64-encoded credentials, of the form `_json_key:{"type":...}`. I can use curl to access the registry like this: `curl -v -H "Authorization: Basic $(cat gcr.b64)" https://gcr.io/v2/my-project/test/manifests/1`. I've used renovate's encryption to add those credentials, but renovate says `DEBUG: Failed to look up docker package gcr.io/my-project/test`.

### Logs (if relevant)

<details><summary>Logs</summary>

```
DEBUG: Repository config
{
  "fileName": ".github/renovate.json"
  "config": {
    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
    "extends": [
      "config:recommended"
    ],
    "ignorePaths": [
      "apps/*/base/charts/**",
      "apps/*/mixins/*/charts/**"
    ],
    "hostRules": [
      {
        "matchHost": "gcr.io",
        "authType": "Basic",
        "encrypted": {
          "token": "***********"
        }
      }
    ]
  }
}

DEBUG: Adding token authentication for gcr.io (hostType=undefined) to hostRules

DEBUG: Failed to look up docker package gcr.io/my-project/test
{
  "dependency": "gcr.io/my-project/test"
  "packageFile": "apps/actions-test/base/kustomization.yaml"
}

```

</details>

**Comment by rarkins** [comment]
Your host rule looks correct. Check the http stats to see the exact URL called

**Comment by cjyar** [comment]
Sorry, I don't work at that company anymore, and I don't have access to the logs. @bobwall23, if you want to troubleshoot this, here's the issue.
